test/videojs-http-streaming.test.js

```diff
@@ -4608,6 +4610,8 @@ QUnit.test(
         );
       }
     );
+    // allows ie to start loading segments, from setupFirstPlay
+    this.player.tech_.readyState = () => 4;
```

Comment:
without this segments were not downloading on ie 11 for this test. Due to https://github.com/videojs/http-streaming/blob/38f5860ec0b4540fe9bd5430bd3cc2993c0f8122/src/master-playlist-controller.js#L692